app/views/user_mailer/survey.text.erb

```diff
@@ -1,9 +1,10 @@
+Open Humans is also looking for a web developer who is interested in open science. So if you are looking for a programming job that helps to make a difference, this one might be for you: https://www.openhumans.org/jobs/
 
-The second topic we wanted to share with you is less light, but it is a topic that is close to our heart at openSNP, as it concerns the free sharing of information and knowledge: Diego Gómez, a conservation biologist from Colombia, who is being sued for sharing another researcher’s Master thesis on the web. Diego originally got access to it through a closed Facebook group and made it publicly available to help others studying the biology of amphibians in his hometown university. In 2014 and under Colombia’s strict copyright laws, the author pressed criminal charges against Diego, where a conviction would mean 4-8 years in prison in addition to a fine. This May, Diego was acquitted of all charges in the first instance, by proving that the document was already available online, and having neither the intention to gain financial benefit from it nor to harm the author (https://www.karisma.org.co/compartirnoesdelito/diego-gomez-found-innocent/).
+And if you happen to be in or near Zurich, Switzerland on the weekend of the 2nd and 3rd of September we have a great participatory genetics event for you: Ernst Hafen and Ulrich Genick of the ETH Zurich are running a workshop on the biology and genetics of smell and taste. In the workshop they will cover the current state of research on smell and taste perception, followed by some practical experiments for you. More information can be found on their (German language) website at http://www.scientifica.ch/events/workshops/Erforschen-Sie-als-Citizen-Scientists-Ihren-Geruchs-und-Geschmackssinn/
 
-This could be the end of the story, but the ruling has been appealed and Diego will now have to defend himself in a higher court. To help paying his legal defense, the Fundación Karisma, who has been advising Diego from the start, has started a crowdfunding campaign: https://www.generosity.com/education-fundraising/compartir-no-es-delito-sharing-is-not-a-crime/x/1756182 No one should face years in prison for sharing scientific knowledge. This is why we have already personally contributed to Diego’s defense fund and want to encourage you to spread the word about the campaign (and contribute if you can afford it).
+Until next time,
+Helge & Bastian, for the openSNP-team
```

Comment:
Are the last two lines supposed to be here and not in the HTML version?

Reply:
whops, good catch!